app/routes/quotes/components/Quote.js

```diff
@@ -21,19 +23,26 @@ type Props = {
   displayAdmin: boolean,
   currentUser: any,
   loggedIn: boolean,
-  comments: Object,
-  deleteComment: (id: ID, contentTarget: string) => Promise<*>
+  emojis: Array<EmojiEntity>,
+  addReaction: ({
+    emoji: string,
+    contentTarget: string
+  }) => Promise<*>,
+  deleteReaction: ({ reactionId: ID, contentTarget: string }) => Promise<*>,
+  fetchEmojis: () => Promise<*>,
+  fetchingEmojis: boolean,
+  emojis: Array<EmojiEntity>
 };
 
 type State = {
   deleting: boolean,
-  showComments: boolean
+  showReactions: boolean
```

Comment:
Remove this if https://github.com/webkom/lego-webapp/pull/1622/files#diff-f31e3b34ecfb11e6258937d6ff8c84eaL84 is removed